appveyor.yml

```diff
@@ -16,6 +16,7 @@ deploy:
     - provider: NuGet
       api_key:
           secure: +CD+4G+gMInD/BjNvO24NW9u1udGdsx0fUi6bh0muNezQ9fP0sHnS1f96f0OXUy6
+      artifact: .*\.s?nupkg
```

Comment:
I was wrong, you need to have a `artifact` regex. I think the Appveyor docs are a little confusing, because they suggest that it will automatically publish nuget packages. However, this appears to only be true if you're building a VS solution directly. Instead we're calling `dotnet build` ourselves, so it doesn't apply.